modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -0,0 +1,240 @@
+* *Method Label Head Color*: Set the color of the labels of *HEAD* methods.
+
+== API Console Request Panel
+
+The API console request panel component shows a panel where users can test an API by making requests to it.
+
+* *API*: Choose the API to display.
+* *Show Documentation Pages*: Show the documentation pages for the API.
+* *Method Label Get Background Color*: Set the background color of the labels of *GET* methods.
+* *Method Label Get Color*: Set the color of the labels of *GET* methods.
+* *Method Label Post Background Color*: Set the background color of the labels of *POST* methods.
+* *Method Label Post Color*: Set the color of the labels of *POST* methods.
+* *Method Label Put Background Color*: Set the background color of the labels of *PUT* methods.
+* *Method Label Put Color*: Set the color of the labels of *PUT* methods.
+* *Method Label Delete Background Color*: Set the background color of the labels of *DELETE* methods.
+* *Method Label Delete Color*: Set the color of the labels of *DELETE* methods.
+* *Method Label Patch Background Color*: Set the background color of the labels of *PATCH* methods.
+* *Method Label Patch Color*: Set the color of the labels of *PATCH* methods.
+* *Method Label Options Background Color*: Set the background color of the labels of *OPTIONS* methods.
+* *Method Label Options Color*: Set the color of the labels of *OPTIONS* methods.
+* *Method Label Head Background Color*: Set the background color of the labels of *HEAD* methods.
+* *Method Label Head Color*: Set the color of the labels of *HEAD* methods.
+* *Link Hover Color*: Set the color of links when the cursor hovers over them.
+* *Button Text Color (Optional)*: Set the color of text on buttons.
+* *Button Hover Text Color (Optional)*: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
+* *Headline Font Family (Optional)*: Set the font family of the headline.
+* *Error Color (Optional)*: Set the color of error messages.
+
+== API Documentation Viewer
+
+The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+
+* *API*: Choose the API.
+* *Section (Optional)*: Set the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+* *Selected Page (Optional)*: Set the page to open.
+
+== API Header
+
+The API header component displays the API Header including icon, name, and description.
+
+* *API*: Choose the API to show.
+
+== Application Analytics Graph
+
+The application analytics graph component displays statistics about the use of an application.
+
+* *Application*: Choose the application to show.
+
+== Application Details
+
+The application details component displays details about an application.
+
+// check component description
+
+* *Application*: Choose the API to show.
+
+== Application Listing
```

Comment:
The values below sound as if they apply only to the carousel; is that correct?

Reply:
Each application is on a card, but I'm not sure if the cards are grouped in a carousel similar to the API carousel or just a grid that is made long enough to show all applications.